editor/src/dev/load_project.clj

```diff
@@ -157,25 +155,14 @@
       (project/make-project project-graph-id workspace extensions))))
 
 (defonce node-load-infos
-  (let [node-load-infos
-        (when (run-task? :read-resources)
-          (dev/run-with-progress "Reading Files..."
-            (fn read-resources-with-progress [render-progress!]
-              (measure-task!
-                :read-resources
-                (project/read-node-load-infos node-id+resource-pairs render-progress! resource-metrics)))))
-
-        node-load-infos
-        (run-and-measure-task!
-          :sort-source-values
-          (project/sort-node-load-infos-for-loading node-load-infos {} {}))]
-
-    (run-and-measure-task!
-      :store-source-values
-      (project/store-loaded-disk-sha256-hashes! node-load-infos workspace)
-      (project/store-loaded-source-values! node-load-infos))
-
-    node-load-infos))
+  (when (run-task? :read-resources)
+    (dev/run-with-progress "Reading Files..."
+      (fn read-resources-with-progress [render-progress!]
+        (measure-task!
+          :read-resources
+          (project/read-nodes node-id+resource-pairs
+            :render-progress! render-progress!
+            :resource-metrics resource-metrics))))))
```

Comment:
`project/read-nodes` now does the sorting internally, and `project/load-nodes!` stores the source values for the `node-load-infos` you give it.